packages/components/src/Card/tests/CardClickable.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { CardClickable } from "../CardClickable";
+
+test("it should trigger the click event after enter press", () => {
+  const clickHandler = jest.fn();
+
+  const { getByTestId } = render(
+    <CardClickable className="boink" onClick={clickHandler}>
+      <p>Press enter</p>
+    </CardClickable>,
+  );
+
```

Comment:
Could we add like fireEvent.keyUp(getByTestId("clickable-card"), { key: "Backspace" });
expect(clickHandler).toNotHaveBeenCalledTimes();

just to make sure it is not being called on any other key press.